### ISSUE TITLE
PHB, DMG, MM need new context

### ISSUE THREAD

**Issue by hyphonetics** [petrgon/beyond-my-content#4]
The PHB, DMG, MM, and BR all have updated context and need to be amended for the script to work.
Thankfully it's as easy as changing

```
"filter-source-basic-rules"
```
to 
```
"filter-source-basic-rules-2014"
```

hopefully in a couple days when the new sourcebooks get released we can add a "2024" suffix to the code to filter those books as well

**Comment by petrgon** [comment]
I've added some other sources too. Don't forget to backup the `mySources` variable before updating.

**Comment by petrgon** [comment]
Added 5e2024 books. Released in 2.3.2.
